File: freelance_market_place/backend/src/utils/password.py
from pwdlib import PasswordHash

password_hash = PasswordHash.recommended()

def hash_password(password : str):
    hashed = password_hash.hash(password=password)
    return hashed


def check_password(password : str, hashed_password: str):
    check = password_hash.verify(password=password, hash=hashed_password)
    return check

File: LiveSupportChatSystem/backend/src/utils/password.ts
import bcrypt from "bcrypt"

export const hashPassword = async (password: string): Promise<string> => {
    const hashed_password = await bcrypt.hash(password, 10);
    return hashed_password
};

export const comparePassword = async (password: string, hash: string): Promise<boolean> => {
    const compared_password = await bcrypt.compare(password, hash);
    return compared_password
};
